cmd/run/run.go

```diff
@@ -150,6 +150,8 @@ func NewRunCommand() *cobra.Command {
 
 	flags.String("trace-otlp-endpoint", defaultConfig.Trace.OTLP.Endpoint, "the endpoint of the trace collector")
 
+	flags.Bool("trace-otlp-insecure", defaultConfig.Trace.OTLP.Insecure, "use insecure connection for trace collector")
```

Comment:
Please add this to the `config.schema.json` file as well 👍

Reply:
I am sorry, I can't find that file. Is it supposed to be in this repository?

Reply:
Haha, thanks! My editor ignored the file as a dotfile 🤦